### PR TITLE
fix: cleanup safari bugs

### PR DESCRIPTION
**Pull request by hwkr**
I think this should fix everything?

### CODE REVIEW

File: packages/api-client/src/helpers/findVariables.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const findVariables = (value: string) => {
   // Wrapped in single or double curly braces.
   // Ignores whitespace
   // Works with lowercase, uppercase, numbers, dashes, underscores
-  const regex = /(?<=\{)\s*\w+\s*(?=\})/g
+  const regex = /(?=\{)\s*\w+\s*(?=\})/g
```
Comment (hwkr) [inline packages/api-client/src/helpers/findVariables.ts:9]: So safari doesn't support look behind until version 16.4 and I was running 16.3, do we need this?

https://caniuse.com/js-regexp-lookbehind

File: packages/api-reference/src/components/Content/ReferenceEndpoint/ExampleRequest.vue [hunk 2]
```diff
@@ -301,7 +301,7 @@ computed(() => {
   padding: 6px;
 }
 .request-editor-section {
-  display: grid;
+  display: flex;
```
Comment (hwkr) [inline packages/api-reference/src/components/Content/ReferenceEndpoint/ExampleRequest.vue:304]: I _think_ this fixes DOC-1444?

Comment (cameronrohani) [reply packages/api-reference/src/components/Content/ReferenceEndpoint/ExampleRequest.vue:304]: it did for me!

**Review by marclave — APPROVED** [review]
awesome work @hwkr ✨

**Pull request merged**